File: SampleCode/RestApi/Clients/Java/src/main/java/io/swagger/client/model/PlaystateCommand.java
/*
 * Emby Server REST API
 * 
 */

package io.swagger.client.model;

import java.util.Objects;
import java.util.Arrays;
import io.swagger.v3.oas.annotations.media.Schema;
import com.google.gson.annotations.SerializedName;
import java.io.IOException;
import com.google.gson.TypeAdapter;
import com.google.gson.annotations.JsonAdapter;
import com.google.gson.stream.JsonReader;
import com.google.gson.stream.JsonWriter;

/**
 * Enum PlaystateCommand  
 */
@JsonAdapter(PlaystateCommand.Adapter.class)
public enum PlaystateCommand {
  STOP("Stop"),
  PAUSE("Pause"),
  UNPAUSE("Unpause"),
  NEXTTRACK("NextTrack"),
  PREVIOUSTRACK("PreviousTrack"),
  SEEK("Seek"),
  REWIND("Rewind"),
  FASTFORWARD("FastForward"),
  PLAYPAUSE("PlayPause"),
  SEEKRELATIVE("SeekRelative");

  private String value;

  PlaystateCommand(String value) {
    this.value = value;
  }

  public String getValue() {
    return value;
  }

  @Override
  public String toString() {
    return String.valueOf(value);
  }

  public static PlaystateCommand fromValue(String input) {
    for (PlaystateCommand b : PlaystateCommand.values()) {
      if (b.value.equals(input)) {
        return b;
      }
    }
    return null;
  }

  public static class Adapter extends TypeAdapter<PlaystateCommand> {
    @Override
    public void write(final JsonWriter jsonWriter, final PlaystateCommand enumeration) throws IOException {
      jsonWriter.value(String.valueOf(enumeration.getValue()));
    }

    @Override
    public PlaystateCommand read(final JsonReader jsonReader) throws IOException {
      Object value = jsonReader.nextString();
      return PlaystateCommand.fromValue((String)(value));
    }
  }
}

File: SampleCode/RestApi/Net.RestSharp/EmbyClient.Dotnet/Model/PlaystateCommand.cs
/*
 * EmbyClient.Dotnet
 */

using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Text.RegularExpressions;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using SwaggerDateConverter = EmbyClient.Dotnet.Client.SwaggerDateConverter;

namespace EmbyClient.Dotnet.Model
{
    /// <summary>
    /// Enum PlaystateCommand  
    /// </summary>
    /// <value>Enum PlaystateCommand  </value>
    [JsonConverter(typeof(StringEnumConverter))]
        public enum PlaystateCommand
    {
        /// <summary>
        /// Enum Stop for value: Stop
        /// </summary>
        [EnumMember(Value = "Stop")]
        Stop = 1,
        /// <summary>
        /// Enum Pause for value: Pause
        /// </summary>
        [EnumMember(Value = "Pause")]
        Pause = 2,
        /// <summary>
        /// Enum Unpause for value: Unpause
        /// </summary>
        [EnumMember(Value = "Unpause")]
        Unpause = 3,
        /// <summary>
        /// Enum NextTrack for value: NextTrack
        /// </summary>
        [EnumMember(Value = "NextTrack")]
        NextTrack = 4,
        /// <summary>
        /// Enum PreviousTrack for value: PreviousTrack
        /// </summary>
        [EnumMember(Value = "PreviousTrack")]
        PreviousTrack = 5,
        /// <summary>
        /// Enum Seek for value: Seek
        /// </summary>
        [EnumMember(Value = "Seek")]
        Seek = 6,
        /// <summary>
        /// Enum Rewind for value: Rewind
        /// </summary>
        [EnumMember(Value = "Rewind")]
        Rewind = 7,
        /// <summary>
        /// Enum FastForward for value: FastForward
        /// </summary>
        [EnumMember(Value = "FastForward")]
        FastForward = 8,
        /// <summary>
        /// Enum PlayPause for value: PlayPause
        /// </summary>
        [EnumMember(Value = "PlayPause")]
        PlayPause = 9,
        /// <summary>
        /// Enum SeekRelative for value: SeekRelative
        /// </summary>
        [EnumMember(Value = "SeekRelative")]
        SeekRelative = 10    }
}
